crates/sparrow-runtime/src/metadata/raw_metadata.rs

```diff
@@ -21,43 +41,117 @@ pub struct RawMetadata {
 }
 
 impl RawMetadata {
-    pub fn try_from(source_path: &Path) -> anyhow::Result<Self> {
+    /// Createa a `RawMetadata` from a `v1alpha::file_path::Path`.
```

Comment:
"create"

Also, are there docs that explain the difference between `RawMetadata` and `TableMetadata` (or whatever the other one used often is)?

Reply:
This is documented in the rust doc for `table_schema` ([here](https://github.com/kaskada-ai/kaskada/blob/main/crates/sparrow-runtime/src/metadata/raw_metadata.rs#L20))